CallsheetLocalizations/it.xcloc/Localized Contents/it.xliff

```diff
@@ -118,70 +128,87 @@
       </trans-unit>
       <trans-unit id="All" xml:space="preserve">
         <source>All</source>
+        <target state="translated">Tutto</target>
         <note/>
       </trans-unit>
       <trans-unit id="All Messages" xml:space="preserve">
         <source>All Messages</source>
+        <target state="translated">Tutti i Messaggi</target>
         <note>Shown in the context of a semi-hidden debugging screen.</note>
       </trans-unit>
       <trans-unit id="App Icon" xml:space="preserve">
         <source>App Icon</source>
+        <target state="translated">Icona</target>
         <note/>
       </trans-unit>
       <trans-unit id="Are you sure?" xml:space="preserve">
         <source>Are you sure?</source>
+        <target state="translated">Sei sicuro/a?</target>
         <note>Shown when clearing recent searches or search history</note>
       </trans-unit>
       <trans-unit id="As rated by users of" xml:space="preserve">
         <source>As rated by users of</source>
+        <target state="translated">Come valutato dagli utenti di</target>
         <note>Shown in a popover explaining TV/movie scores</note>
       </trans-unit>
       <trans-unit id="Average runtime" xml:space="preserve">
         <source>Average runtime</source>
+        <target state="translated">Durata media</target>
         <note>How long episodes run in a single season of TV</note>
       </trans-unit>
       <trans-unit id="Be a hero and support independent apps" xml:space="preserve">
         <source>Be a hero and support independent apps</source>
+        <target state="translated">Sii un eroe e supporta app indipendenti</target>
         <note>Shown when trying to sell a subscription</note>
       </trans-unit>
       <trans-unit id="Be like the many other beautiful people who have rated this version." xml:space="preserve">
         <source>Be like the many other beautiful people who have rated this version.</source>
+        <target state="translated">Sii come le tante altre persone meravigliose che hanno valutato questa versione</target>
         <note>Shown to try to encourage the user to rate the app</note>
       </trans-unit>
       <trans-unit id="Began Airing" xml:space="preserve">
         <source>Began Airing</source>
+        <target state="translated">Inizio trasmissione</target>
         <note>When a TV season started airing</note>
       </trans-unit>
       <trans-unit id="Begins Airing" xml:space="preserve">
         <source>Begins Airing</source>
+        <target state="translated">La trasmissione inizierà</target>
         <note>When a TV season will start airing</note>
       </trans-unit>
       <trans-unit id="Biography" xml:space="preserve">
         <source>Biography</source>
+        <target state="translated">Biografia</target>
         <note/>
       </trans-unit>
       <trans-unit id="Born" xml:space="preserve">
         <source>Born</source>
+        <target state="translated">Data di nascita [or Nato/a]</target>
```

Comment:
Is this a question about gender, I assume? If there is an ungendered way to have a default, I'd prefer that, but if we have to pick one, let's go with masculine. I need to work on making this better in the future.

Reply:
Yes, it was. _Data di nascita_ is gender neutral, I'll commit the changes appropriate for all your comments above, including this one, right away.

Reply:
You're going to run into this one a lot in Romance languages (SP/FR/IT/PT etc). Here, it's clearly about a person, so using that term seems "gender neutral" in that sense. Just means "Date of Birth".

But the term is in the feminine because "data" is feminine. That's why "nascita" ends with and "a". This is called "agreement" and most of the time it means nothing loaded, it's just based on the sound of the word. These languages don't have as many arguments about terms like "hey guys" because everything is gendered and it's pretty meaningless most of the time (not all of the time).

So, you'll get things like this:

"the purple boats" -> "les bateaux violets" (masculine)
"the purple cars" -> "les voitures violettes" (feminine)

The way you describe the color changes based on an arbitrary gender assignment. I don't know which langauges everyone knows, so sorry if that's too much explanation.

https://blog.duolingo.com/mistakes-in-french-for-learners/#2-french-nouns-gender-agreement

Reply:
Yeah, I'm not really sure how to skin this cat, to be completely honest. I think asking translators to use gender-neutral where possible, and gendered where it's just weird/dumb/wrong to be neutral, is fine with me.

Reply:
That seems reasonable. Just watch out for people with an axe to grind about this stuff. (I don't care, I am a descriptivist)

Reply:
The first link is a sparkler:

https://blog.duolingo.com/what-is-grammatical-gender/

It glosses over one issue, which is that you refer to a mixed-gender group of people in the masculine. If it's all ladies, you can use the feminine plural. That is where most conflict happens, but it's more difficult to detach it from the language, as you can in English.